app/services/jokes.js

```diff
@@ -0,0 +1,21 @@
+const axios = require('axios');
+
+const { externalServiceError } = require('../errors');
+const config = require('../../config');
+
+const { baseUrl } = config.common.geekJokesApi;
```

Comment:
Sorry, I forgot this before.
You can do something like this:
`const {common: { geekJokesApi: { baseUrl } } } = require('../../config');`